srcpkgs/liblcf/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'liblcf'
+pkgname=liblcf
+version=0.6.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config autoconf automake libtool"
+makedepends="expat-devel icu-devel"
+short_desc="Library to handle RPG Maker 2000/2003 and EasyRPG projects"
+maintainer="Franc[e]sco <lolisamurai@tfwno.gf>"
+license="MIT"
+homepage="https://github.com/EasyRPG/liblcf"
+distfiles="https://github.com/EasyRPG/liblcf/archive/${version}.tar.gz"
+checksum=bb1848af7229071268d5f543b96837fefcd2dfc109a348cbd38169b4195257f5
+
+pre_configure() {
+	autoreconf -i
+}
+
+post_install() {
+	vlicense COPYING
+}
+
+liblcf-devel_package() {
+	short_desc+=" - development files"
+	depends="${makedepends} ${sourcepkg}>=${version}_${revision}"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/pkgconfig
+		vmove "usr/lib/*.a"
+		vmove "usr/lib/*.so"
+		vmove usr/lib/cmake
+		vmove usr/share
```

Comment:
be careful with moving the whole of usr/share as it can end up moving assets meant for the mainpkg when updates are done.